mesheryctl/internal/cli/root/app/app.go

```diff
@@ -42,10 +43,27 @@ var AppCmd = &cobra.Command{
 // Base command
 mesheryctl app [subcommand]
 	`,
-	RunE: func(cmd *cobra.Command, args []string) error {
-		if len(args) == 0 {
-			return cmd.Help()
+	PersistentPreRunE: func(cmd *cobra.Command, args []string) error {
```

Comment:
This code is repeated multiple times, will it be possible to move it to utils package and all other mesheryctl sub commands make use of this function ?

Reply:
@hexxdump  done